### PR TITLE
prow/config: support tidb feature branch distribute-reorg

### PR DESCRIPTION
**Pull request by zimulala**
support tidb feature branch: https://github.com/pingcap/tidb/tree/distribute-reorg

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @purelind

**Comment by zimulala** [comment]
PTAL @wuhuizuo

**Review by wuhuizuo — APPROVED** [review]

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -1622,6 +1622,21 @@ ti-community-format-checker:
...
+          branches:
+            - master
+            - feature/distribute-reorg
+          regexp: "(?im)^Issue Number:\\s*((,\\s*)?(ref|close[sd]?|resolve[sd]?|fix(e[sd])?)\\s*((https|http)://github\\.com/{{.Org}}/{{.Repo}}/issues/|{{.Org}}/{{.Repo}}#|#)(?P<issue_number>[1-9]\\d*))+"
+          missing_message: |
+            **Notice**: To remove the `do-not-merge/needs-linked-issue` label, please provide the linked issue number on one line in the PR body, for example: `Issue Number: close #123` or `Issue Number: ref #456`.
+
+            <sub>:open_book: For more info, you can check the ["Contribute Code"](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/contribute-code.html#referring-to-an-issue) section in the development guide.</sub>
+          missing_label: do-not-merge/needs-linked-issue
+          skip_label: skip-issue-check
```
Comment (wuhuizuo) [inline prow/config/external_plugins_config.yaml:1639]: 缩进有问题。

**Review by wuhuizuo — CHANGES_REQUESTED** [review]

**Review by wuhuizuo — APPROVED** [review]

**Pull request merged**